src/util/newconfig/ConfigConstraints.hpp

```diff
@@ -78,7 +78,7 @@ static constexpr std::array<char const*, 3> kLOAD_CACHE_MODE = {
 /**
  * @brief specific values that are accepted for database type in config.
  */
-static constexpr std::array<char const*, 2> kDATABASE_TYPE = {"cassandra", "scylladb"};
+static constexpr std::array<char const*, 1> kDATABASE_TYPE = {"cassandra"};
```

Comment:
This should not be changed.. i imagine someone out there uses `scylladb` as their DB setting and this will break their configuration.